tests/run_qsim_test.cc

```diff
@@ -62,7 +62,7 @@ R"(4
 10 h 3
 )";
 
-TEST(QSimRunner, RimQSim) {
+TEST(QSimRunner, RimQSim1) {
```

Comment:
Should this be `RunQSim1`? Similarly for the other test.

Reply:
Yes, thank you!